Directory.Packages.props

```diff
@@ -23,6 +23,7 @@
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)' != 'net48'">
+		<PackageVersion Include="Microsoft.Win32.SystemEvents" Version="5.0.0" />
```

Comment:
As per comment on `ArchiSteamFarm.csproj`

Reply:
Not critical for this PR.